src/components/home/elements/TokenRecord.vue

```diff
@@ -274,16 +276,26 @@ export default {
             } else return ''
         },
         isCertified() {
-            return certify.isCertified(this.tokenId)
+            return function(tokenId) {
```

Comment:
If it is required to pass parameters, change to **method**, not **computed**. 
Move `isCertified(tokenId)`, `officialName(tokenId)` and `officialTokenSvg(tokenId)` to **method**.